ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_creation.py

```diff
@@ -551,6 +551,13 @@ def test_jax_numpy_full(
     )
 
 
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.double",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("double"),
```

Comment:
The function double casts arrays into a double (float64) data type, generating double values only would not test the function properly, we should instead be generating any dtype that is castable into a double (float64), that is any numeric type actually.